docs/lib/debugging/fragments/android/dev-menu/usage.md

```diff
@@ -0,0 +1,6 @@
+To access the developer menu during a debug build of your app, shake the device. An emulator can be 
+shaken by selecting the more option (denoted by three dots) next to the emulator, navigating to the 
+“Virtual Sensors” tab, clicking the “Move” button, and then moving the emulator shown within the 
+“Virtual Sensors” tab back and forth a few times.
+
+![Shake Android Emulator](~/images/debugging/shakeAndroidEmulator.gif)
```

Comment:
1.8 MB is quite large. Any chance, we can reduce the size? (lower frame rate, resolution, etc.)

Reply:
I generated this and wasn't sure. Any tool you can recommend to do that?